dashboard/settings.py

```diff
@@ -496,6 +486,17 @@ def show_debug_toolbar(request):
 SESSION_COOKIE_SAMESITE = ENV.get("SESSION_COOKIE_SAMESITE", None)
 CSRF_COOKIE_SAMESITE = ENV.get("CSRF_COOKIE_SAMESITE", None)
 
+ENABLE_BACKEND_LOGIN = False if STUDENT_DASHBOARD_SAML or STUDENT_DASHBOARD_LTI else True 
+
+# Allow for ENABLE_BACKEND_LOGIN override
+ENABLE_BACKEND_LOGIN = ENV.get("ENABLE_BACKEND_LOGIN", ENABLE_BACKEND_LOGIN)
```

Comment:
I felt `ENABLE_BACKEND_LOGIN` on L489 can be written as `CHECK_TO_ENABLE_BACKEND_LOGIN` since that is actually checking if there is need to enable it or not and actually the action of enabling it is happening on L492

Reply:
Yeah, that's just figuring out the default, I can rename that initial variable.